routes/api/account.js

```diff
@@ -253,6 +253,7 @@ module.exports = {
             Middleware.Account.parsePatch,
 
             Middleware.Account.updateAccount,
+            Middleware.Auth.sendConfirmAccountEmailMiddleware,
```

Comment:
can we change this so the word middleware isn't there twice? like to `Middleware.Auth.sendConfirmAccountEmail`? or maybe even shorter bc this is really long